Assets/MRTK/SDK/Experimental/Drawstring/Scripts/TwoHandleManipulator.cs

```diff
@@ -0,0 +1,937 @@
+            get => twoHandedManipulationType;
+            set => twoHandedManipulationType = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Specifies whether manipulation can be done using far interaction with pointers.")]
+        private bool allowFarManipulation = true;
+
+        /// <summary>
+        /// Specifies whether manipulation can be done using far interaction with pointers.
+        /// </summary>
+        public bool AllowFarManipulation
+        {
+            get => allowFarManipulation;
+            set => allowFarManipulation = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Check to enable frame-rate independent smoothing.")]
+        private bool smoothingActive = true;
+
+        /// <summary>
+        /// Check to enable frame-rate independent smoothing.
+        /// </summary>
+        public bool SmoothingActive
+        {
+            get => smoothingActive;
+            set => smoothingActive = value;
+        }
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the movement. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float moveLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the movement. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float MoveLerpTime
+        {
+            get => moveLerpTime;
+            set => moveLerpTime = value;
+        }
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the rotation. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float rotateLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the rotation. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float RotateLerpTime
+        {
+            get => rotateLerpTime;
+            set => rotateLerpTime = value;
+        }
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the scale. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float scaleLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the scale. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float ScaleLerpTime
+        {
+            get => scaleLerpTime;
+            set => scaleLerpTime = value;
+        }
+
+        /// <summary>
+        /// Returns true when one or more handles are being interacted with.
+        /// </summary>
+        public bool IsGrabbed => pointerIdToPointerMap.Count > 0;
```

Comment:
I don't think that these two properties are serialized fields right?

Reply:
i.e. they are in the serialized fields region

Reply:
Ah, you're right.